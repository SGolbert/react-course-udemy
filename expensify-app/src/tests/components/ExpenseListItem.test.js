import React from "react";
import { shallow } from "enzyme";
import ExpenseListItem from "../../components/ExpenseListItem";
import expenses from "../fixtures/expenses";

test("should render ExpenseListItem with the first fixture expense", () => {
  const expense = expenses[0];
  const wrapper = shallow(<ExpenseListItem {...expense} key={expense.id} />);
  expect(wrapper).toMatchSnapshot();
});
