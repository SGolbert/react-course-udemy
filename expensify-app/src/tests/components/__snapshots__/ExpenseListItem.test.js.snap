// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with the first fixture expense 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    195
     Cents - 
    0
  </p>
</div>
`;
